css/style.css

```diff
@@ -1,11 +1,227 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+header {
+  display: flex;
+  padding: 40px;
+  margin-left: 100px;
+}
+
+/* Main Image*/
+.main_image {
+  margin-bottom: 600px;
+  position: relative;
+  top: -70px;
+}
+
+*/*Top Left Karma Logo*/
+.header__logo {
+  height: 50px;
+  margin-right: 900px;
```

Comment:
This is pushing the nav all the way off the screen. Remove it, and instead try adding `justify-content: space-between;` to the parent (`header`)